int/api/myplugin/plugin.go

```diff
@@ -31,8 +31,9 @@ const (
 
 	errorCodePluginInstallationInvalidSource = "Plugin-0010"
 
-	errorCodePluginRegisterUnknown     = "Plugin-0020"
-	errorCodePluginRegisterInvalidData = "Plugin-0020"
+	errorCodePluginRegisterUnknown           = "Plugin-0020"
+	errorCodePluginRegisterInvalidData       = "Plugin-0020"
+	errorCodePluginRegisterAlreadyRegistered = "Plugin-0020"
```

Comment:
Those error should have different identifiers since it should allow us to handle the error and print the right error message in the front end.
```suggestion
	errorCodePluginRegisterUnknown           = "Plugin-0020"
	errorCodePluginRegisterInvalidData       = "Plugin-0021"
	errorCodePluginRegisterAlreadyRegistered = "Plugin-0022"
```